src/project/cache/flowr-analyzer-cache.ts

```diff
@@ -185,10 +185,19 @@ export class FlowrAnalyzerCache<Parser extends KnownParser> extends FlowrCache<A
 		if(useDataflow) {
 			/* if force is active, it will have triggered with normalize */
 			dataflow = await this.dataflow();
+		} else {
+			dataflow = this.peekDataflow();
+		}
+
+		let result: ControlFlowInformation;
+		if(useDataflow || simplifications.length > 0) {
+			result = extractCfg(normalized, this.args.config, dataflow?.graph, simplifications);
+		} else if(dataflow && dataflow.cfgQuick) {
+			result = dataflow.cfgQuick;
```

Comment:
can we mirror this fact into the cache for this? Otherwise it feels harder to write the bootstrapping for the cfgs with additional passing.

Reply:
Extra PR, we do this once we start with the incremental cfg system

Reply:
We now store the fact that this is a "quick CFG" in the cache.
The incremental CFG system is described in #1963.